OJP/OJP_JourneySupport.xsd

```diff
@@ -819,24 +819,24 @@
 		<xs:sequence>
 			<xs:element name="TrackSection" type="TrackSectionStructure" maxOccurs="unbounded">
 				<xs:annotation>
-					<xs:documentation>LINK PROJECTION on the infrastructure network of the leg section together with time information.</xs:documentation>
+					<xs:documentation>An aggregate of information that may be leaning on LEG TRACK together with duration and length that can be used for representing the leg on a map. The points may be STOP PLACE, SCHEDULED STOP POINT, coordinates or ADDRESSes. Specialisation of SECTION.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
 		</xs:sequence>
 	</xs:complexType>
 	<xs:complexType name="TrackSectionStructure">
 		<xs:annotation>
-			<xs:documentation>LINK PROJECTION on the infrastructure network of the LEG together with time information</xs:documentation>
+			<xs:documentation>SECTION of a LEG TRACK on the infrastructure network of the LEG together with time information.</xs:documentation>
 		</xs:annotation>
 		<xs:sequence>
 			<xs:element name="TrackStart" type="PlaceRefStructure" minOccurs="0">
 				<xs:annotation>
-					<xs:documentation>Start place of this track.</xs:documentation>
+					<xs:documentation>Start place of this track (Start of a SECTION of a LEG.TRACK).</xs:documentation>
```

Comment:
Wouldn't *SectionStart* and *SectionEnd* be more appropriate than *TrackStart/End*?

Reply:
changed